worlds/generic/docs/RunFromSourceGuideForMac.md

```diff
@@ -0,0 +1,23 @@
+# Prerequisite Software
+Here is a list of software to install and source code to download.
+1. [Python](https://www.python.org/downloads/macos/), 3.8 or newer is necessary.
+2. [Xcode](https://apps.apple.com/us/app/xcode/id497799835)
+3. [Archipelago Source](https://github.com/ArchipelagoMW/Archipelago/releases), download the source code in either .zip or tar.gz formats of the newest version.
+4. [SNI](https://github.com/alttpo/sni/releases), be sure to select the asset with 'darwin' in the name. This one is only necessary for SNES games (A Link to the Past, Super Metroid, and SMZ3 at time of writing).
+5. If you would like to generate Enemized seeds for ALTTP locally (not on the website), you will need the EnemizerCLI at this [link](https://github.com/Ijwu/Enemizer/releases).
+6. An Emulator of your choosing for games that need an emulator. For SNES games, I recommend [RetroArch](https://www.retroarch.com/?page=platforms), entirely because it was the easiest for me to setup on Mac.
+# Steps to Run the Clients
+1. Click on the Archipelago source code zip file to extract the files to an Archipelago directory.
+2. Move this Archipelago directory out of your downloads directory.
+3. Open terminal and navigate to your Archipelago directory.
+5. If your game doesn't have a patch file, run the command `python3 'client of choice'.py`.
+6. If your game does have a patch file, move the base rom to the Archipelago directory and run the command `python3 'client of choice'.py 'path to patch file'` with the filename extension for the patch file (apsm, aplttp, apsmz3) included.
```

Comment:
Document updated with a more valid example command.

Reply:
For the benefit of Linux users running from source and taking hints from this guide, might you write `SNIClient.py` rather than `sniclient.py`?

Thank you by the way, Bicolour. You're generous to share this work and thoughtful to take feedback. I think many people will benefit from it!

Reply:
I don't know how I forgot the actual filename had capital letters, oops. It is now SNIClient.py, and I hope this will help some people out too!